start.sh

```diff
@@ -3,5 +3,6 @@
 mkdir -p ./volumes
 mkdir -p ./volumes/postgres ./volumes/geth ./volumes/zksync/env/dev ./volumes/zksync/data
 
+docker-compose pull
```

Comment:
No-no-no! `pull` should be a part of `clear.sh` script.
If someone has a perfectly working though outdated setup, accidental `pull` may break stuff for them.
We don't want such side effects.

Reply:
Good point!